src/client/game.cpp

```diff
@@ -4063,27 +4065,6 @@ void Game::readSettings()
  ****************************************************************************/
 /****************************************************************************/
 
-void Game::extendedResourceCleanup()
```

Comment:
note: this function is pretty useless, it's a 7 years old function to do some debug